types/office-js-preview/index.d.ts

```diff
@@ -74513,58 +74196,51 @@ declare namespace Word {
      * Underline types
      *
      * [Api set: WordApi]
+     * @remarks
+     * The supported styles for underline format. Must have same value as in word\inc\props.h
```

Comment:
I don't think we want to refer to an internal *.h file in our public docs. 
```suggestion
     * The supported styles for underline format. 
```